### ISSUE TITLE
Bug: unable to view primer at 200% on a laptop screen

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#121]
When trying to replicate bug #120 I noticed that on my laptop screen, when viewing the WCAG Primer at 200% zoom, the menu covered the entire page and I was unable to scroll to read any content on the page.

This fails Web Content Accessibility Guidelines criterion [1.4.4 Resize Text](https://www.w3.org/WAI/WCAG22/Understanding/resize-text) as content is lost.

<img src="https://github.com/user-attachments/assets/41696cfa-3636-4d9b-80ff-a8660a828c6a" alt="WCAG 2.1 Primer menu taking up the entire screen, with options such as Overview, All requirements and Content">

I'm using Chrome on a Windows machine with the operating system size set at 125% (recommended), and a screen resolution of 1920 x 1080 (including browser tabs/address/bookmarks and taskbar)

**Comment by andrewhick** [comment]
I get similar behaviour on a 14-inch Mac. A small amount of text is visible below the menu but it is still impossible to read the page content without changing the zoom level from 200%.
